### PR TITLE
Add speedify latest

### PR DESCRIPTION
**Pull request by numbermaniac**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

### CODE REVIEW

**Comment by vitorgalvao** [comment]
From the contents of the dmg, it seems like uninstallation might be a little more complex. The app bundle even has a `uninstall-speedify-service.sh` script that has:

```shell
#!/bin/sh

launchctl unload /Library/LaunchDaemons/me.connectify.SMJobBlessHelper.plist
rm -f /Library/LaunchDaemons/me.connectify.SMJobBlessHelper.plist
#rm /var/run/SpeedifyService.socket
rm -f /Library/PrivilegedHelperTools/me.connectify.SMJobBlessHelper

launchctl unload /Library/LaunchDaemons/SwitchboardService.plist
rm -f /Library/LaunchDaemons/SwitchboardService.plist
rm -f /var/run/SwitchboardService.socket
rm -f /Library/PrivilegedHelperTools/SwitchboardService
rm -rf /tmp/tuntaposx-*.kext

launchctl unload /Library/LaunchDaemons/SpeedifyService.plist
rm -f /Library/LaunchDaemons/SpeedifyService.plist
rm -f /var/run/SpeedifyService.socket
rm -f /Library/PrivilegedHelperTools/SpeedifyService
```

**Comment by Numbermaniac** [comment]
Neat. Should I change the `uninstall` stanza to just run that script?

**Comment by vitorgalvao** [comment]
Scripts should be done as an extra, not as the only way (when possible).

Add the script, yes, but also extend your `launchctl:`.

**Comment by Numbermaniac** [comment]
I think it's done, please check if it's right.

File: Casks/speedify.rb [hunk 1]
```diff
@@ -8,7 +8,12 @@
 
   app 'Speedify.app'
 
-  uninstall launchctl: 'me.connectify.SMJobBlessHelper'
+  uninstall launchctl: [
+                         'me.connectify.SMJobBlessHelper',
+                         'SwitchboardService',
+                         'SpeedifyService',
+                       ],
+            script:    '/Applications/Speedify.app/Contents/Resources/uninstall-speedify-service.sh'
```
Comment (vitorgalvao) [inline Casks/speedify.rb]: `script:    "#{appdir}/Speedify.app/Contents/Resources/uninstall-speedify-service.sh"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by Numbermaniac** [comment]
Well, looks like you've just done that for me! Anything else need to be done?

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
> Well, looks like you've just done that for me!

I did.

> Anything else need to be done?

No. I was just waiting for Travis.

**Pull request merged**